indexer_app/management/commands/populatedata.py

```
@@ -0,0 +1,547 @@
+)
+from tokens.models import Token
+
+
+class Command(BaseCommand):
+    help = "Pull donations data from contract & populate donations table."
+
+    def handle(self, *args, **options):
+        # lists
+        LISTS_CONTRACT_ID = "lists.potlock.near"
+        url = f"https://rpc.web4.near.page/account/{LISTS_CONTRACT_ID}/view/get_lists"
+        lists = requests.get(url)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)